### PR TITLE
ADD culture

### PR DESCRIPTION
**Pull request by mikefaille**

### CODE REVIEW

File: culture/main.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# SRE Culture
+
+Requirements (they are intimately related) :
+- Trust. Telling the «truth» should not need courage.
```
Comment (elafontaine) [inline culture/main.md]: "Telling the "truth" should not need courage and should be encouraged."  

Everyone is allowed their opinion, but it's good if only the truth gets encouraged, not the fake news.

File: culture/main.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Work oriented value stream. Organisations should care about
```
Comment (elafontaine) [inline culture/main.md]: Not sure what this means... Work oriented needs to be a little bit more defined.

File: culture/main.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- Drive decision using datas.
+- Innovation culture. Working on getting job done is important as working on creating tools and habits to get more time to get job done better and faster.
```
Comment (elafontaine) [inline culture/main.md]: "getting job done is *as* important as working on creating tools and"

File: culture/main.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+- Embracing risk. No systems is perfect. We should care about clients perception. Then don't promise something impossible like having 100% on reliability. For example, when they use cell phone, clients accept less reliability on SaaS  because they know troubles are more frequently their carriers signal. Insted, have at least as an objective  reliability level that clients couldnt make difference between certain level of reliability and 100% reliability.
+
+Embracing risk. No systems is perfect. We should care about clients perception. Then don't promise something impossible like having 100% on reliability. For example, when they use cell phone, clients accept less reliability on SaaS because they know troubles are more frequently their carriers signal. Insted, have at least as an SLO that clients couldnt make difference between certain level of reliability and 100% reliability.
```
Comment (elafontaine) [inline culture/main.md]: twice the same paragraph but different endings.

**Review by elafontaine — APPROVED** [review]

File: culture/main.md (hunk 4 above)
Comment (mikefaille) [reply culture/main.md]: Oups

**Comment by mikefaille** [comment]
@elafontaine Sup

**Pull request merged**